### DISCUSSION TITLE
How to handle mixed language audio?

### DISCUSSION THREAD

**Discussion by AnkS4** [openai/whisper#345] [Q&A]
I would like to transcribe audio file which will be expected to have more than one language.
How do I transcribe it?

One way I could think of is splitting each word in given audio & identify it's language & then transcribe it.
How can I split the audio by words quickly?

**Comment by ejkitchen** [comment]
I would like to echo this request. Would love to be able to handle mixed language audio. In Canada, some of our meetings are bilingual (French/English) and would be nice to handle this automatically. And the meetings are not first half English, second French. There are many meetings where a question is posed in French/English, and the answer is in the other language. Right now we're using diarization to get around this, but it's a bit of a hack, and if two people speak at the same time in different languages then it's very problematic.

**Reply by dschweinbenz** [reply]
Nevertheless the language, a lot of people will use English words when talking in german so transcribing using german should also include english words
